docker.py

```diff
@@ -180,25 +190,31 @@ def get_numpy_choices():
 RUN cd "$PYENV_ROOT" && git pull && cd - && env CFLAGS="-fPIC" PYTHON_CONFIGURE_OPTS="--enable-shared" pyenv install {python_ver}
 RUN pyenv global {python_ver}
 RUN pyenv rehash
+
+ENV CUTENSOR_URL=https://developer.download.nvidia.com/compute/cuda/repos/ubuntu{cutensor_os_ver}/x86_64
```

Comment:
It seems the corresponding line is missing in centos7 configuration.

Reply:
Lets forbit the test with cuTENSOR in centos by now

Reply:
Hmm, Isn't it as simple as doing the similar fix as ubuntu?

Reply:
https://developer.download.nvidia.com/compute/cuda/repos/rhel7/x86_64/

i didnt find a cutensor rpm package for rhel7 (ce ntos7)

Reply:
https://developer.download.nvidia.com/compute/cuda/repos/rhel7/x86_64/

i didnt find a cutensor rpm package for rhel7 (ce ntos7)

Reply:
https://developer.download.nvidia.com/compute/cuda/repos/rhel7/x86_64/

there is a rpm here, let me try it.